5calls/app/src/main/java/org/a5calls/android/a5calls/FiveCallsApplication.java

```diff
@@ -37,6 +38,7 @@
  */
 public class FiveCallsApplication extends Application {
     private int mRunningActivities;
+    public static AnalyticsManager analyticsManager = new AnalyticsManager();
```

Comment:
suggest we have this as a private member with a getter but not a big deal either way.

```
private static AnalyticsManager mAnalyticsManager = ...

public AnalyticsManager analyticsManager() { return mAnalyticsManager; }
```

Reply:
not my favorite android/javaism, but I'll conform 😅